examples/simple_model/preprocessed/simple_model/data/elements/electricity-curtailment.csv

```diff
@@ -0,0 +1,3 @@
+region,name,type,carrier,tech,bus,capacity,marginal_cost,output_parameters
+BE,BE-electricity-curtailment,excess,electricity,curtailment,BE-electricity,0,0,{}
```

Comment:
Capacity should not be zero, should it?

Reply:
I couldn't figure out what's meant with capacity in case of curtailment. So I thought, I set it (the initial capacity?) to zero and the optimizer can expand it.

Reply:
You could just not set capacity, so curtailment is unrestricted.